Microservices/Posts/src/main/java/cs4337/group_8/TemplateProject/services/PostService.java

```diff
@@ -20,10 +20,13 @@ public class PostService {
     @Autowired
     private LikesRepository likesRepository;
 
-    public Post createPost(Post post) {
+    public Post createPost(Post post, String jwtToken) {
+        Long userId = jwtService.extractUserId(jwtToken); // Extract userId from token
```

Comment:
The extractUserId returns Integer, either change the return type or add `Long.valueOf()`